Dustin-Guillen.txt

```diff
@@ -0,0 +1,138 @@
+	An h1 tag is used to identify the main header on a page.
+	An h1 tag should only be used once per page.	
+
+    5. Name two tags that have required attributes
+	The <link>, <a> and <img> tags all require attributes.
+
+    6. What do we put in the head of our HTML document? 
+	The <title>, <link>, <meta>, <style>, <base>, <script> and <noscript> can all go in the head. <title> is the only required element.
+
+    7. What is an id? 
+	An id is and attribute. is is mostly used for styling a specific element. No elements should have the same id.
+
+    8. What elements can I add an id to? 
+	In HTML5 you may ad an id to any element, but it is not useful for all elements.
+
+    9. How many times can I use the same id on a page? 
+	You may only use the same id ONCE per page.
+
+    10. What is a class?
+	 A class is an attribute.It is used to style multiple elements with the same class.
+ 
+    11. What elements can I add a class to? 
+	In HTML5 you may add the class attribute to any element, but it is not useful for all elements.
+
+    12. How many times can I use the same class on a page?
+	 You may use the class attribute for multiple elements on a page. You may use the class elements for as many times as neccesary.
+ 
+    13. How do I get my link to open in a new tab?
+	 Open a link in a new tab by using the target attribute. Use the target atrribute with _blank. target="_blank"
+
+    14. What is the alt attribute used for?
+	 The alt attribute shows text to describe an element that cannot be rendered. The alt attribute is used for screen readers.
+ 
+    15. How do I reference an id?
+	 To reference an id use the # tag before it. #id
+	
+    16. What is the difference between a section and a div?
+	 A <section> is semantic and more specific and easier understood by screen readers, while the <div> is more general.
+
+    17. What is CSS used for? 
+	 Cascading Style Sheets are used to style HTML.
+
+    18. How do we select an element? Example - every h2 on the page
+	 You can select an element is CSS by writing its tag, writing its class with a dot before it, or writing it's id with a hash before it.
+	 Example:
+	 h2
+	 .h2
+	 #h2
+
+    19. What is the difference between a class and an id? - Give me an example of when I might use each one.
+	 The differnece between a class and an id is that classes may be used on multiple elements as many times as needed per page, while id's are used to style ONE specific element.
+	 A class can be used for multiple differnt images in multiple differnt sections. 
+	 An id can be used to style one specific element in a section with multiples of the same element.
+
+    20. How do we select classes in CSS?
+	 You can select a class in CSS by writing the class name with a dot before it. (Example) .class
+
+    21. How do we select a p element with a single class of �human��?
+	 You would select the class of human by writing the class name with a dot before it. (Example) .human
+
```

Comment:
What if multiple different elements had the same tag. I.e what it there was a `div` that had the class of human as well as a `p`? In that case you cant just select for `.human` because you would be targeting both the `div` and the `p`. Gotta be careful!